frontend/src/metabase/components/Triggerable/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { ReactElement } from "react";
+import { isObject } from "underscore";
+
+import { RenderProp } from "./types";
+
+export const isRenderProp = <Props extends Record<string, unknown>>(
+  value: unknown,
+): value is RenderProp<Props> => {
+  return typeof value === "function";
+};
+
+export const isReactElement = (value: unknown): value is ReactElement => {
```

Comment:
I believe these two should live somewhere else (in some kind of "types" directory) since they are very generic.

What would be a good place for them?

Reply:
There is this file https://github.com/metabase/metabase/blob/0e9c50933fbd790291004dbcf333d11d645c5aa2/frontend/src/metabase-types/guards/react.ts#L12. Maybe we could also use `react-is` instead of the custom function?